grails-app/conf/Config.groovy

```diff
@@ -407,9 +407,6 @@ jbrowse {
 //	// Warning: We are still testing the performance of the NeatFeatures plugins in combination with Apollo.
 //	// We advise caution if enabling these plugins with Apollo until this process is finalized.
     plugins {
-        WebApollo {
```

Comment:
@nathandunn No, its because in JBrowseController we are already including `annot.json` which has the `WebApollo` plugin definition.

https://github.com/GMOD/Apollo/blob/2.1.0/grails-app/controllers/org/bbop/apollo/JbrowseController.groovy#L693

With another WebApollo definition in `Config.groovy`, we end up with Apollo client code being initialized twice.

Yes, please test in case I introduced something unintended.